### DISCUSSION TITLE
Unexpected Token in RedirectState

### DISCUSSION THREAD

**Discussion by THEkatinamicrowave** [CodenameCrew/CodenameEngine#472] [Q&A]
![image](https://github.com/user-attachments/assets/b266c711-90d1-4eaf-8eaf-c0812601628b)
it's getting spit from ``import funkin.backend.scripting.events.*``

**Comment by r6915ee** [comment]
> ![image](https://private-user-images.githubusercontent.com/89645987/391299732-b266c711-90d1-4eaf-8eaf-c0812601628b.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzMxMDA3OTgsIm5iZiI6MTczMzEwMDQ5OCwicGF0aCI6Ii84OTY0NTk4Ny8zOTEyOTk3MzItYjI2NmM3MTEtOTBkMS00ZWFmLThlYWYtYzA4MTI2MDE2MjhiLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEyMDIlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMjAyVDAwNDgxOFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWI5NWE5NGMzMTZmZDVlYmE4YTc0OTZhNDU0ZDIzYzJmNGZhODViM2VmMDNkNWQ3ZWM1ZmMwNjExZjU3NDY2YjUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.PSFdCZAr1ZujRPU1wVb59X1OhomoqEFHRZLYcz4x_Kg) it's getting spit from `import funkin.backend.scripting.events.*`

Is it a custom script?

**Reply by THEkatinamicrowave** [reply]
it's StoryMenuState.hx but without the ``package funkin...`` bullshit

**Reply by r6915ee** [reply]
Why would you need a copy of `StoryMenuState.hx` that lacks the package name?

**Reply by THEkatinamicrowave** [reply]
it's modded and I remember seeing an issue that said that removing the package name would fix a bug
![image](https://github.com/user-attachments/assets/54c4d3fe-8820-4d96-9993-4c863bab3757)

**Reply by THEkatinamicrowave** [reply]
![image](https://github.com/user-attachments/assets/d9a26758-8db1-492d-b545-eabfdfb1bf61)

**Reply by r6915ee** [reply]
I think HScript Improved just doesn't recognize wildcards yet. You may try importing all of the files necessary for your script if possible.

**Reply by THEkatinamicrowave** [reply]
rip 😭 
can we implement that

**Reply by r6915ee** [reply]
True dat.

**Reply by NeeEoo** [reply]
no

**Reply by THEkatinamicrowave** [reply]
rip 😭
